cmd/server-startup-msg.go

```diff
@@ -143,7 +145,19 @@ func printServerCommonMsg(apiEndpoints []string) {
 
 // Prints startup message for Object API access, prints link to our SDK documentation.
 func printObjectAPIMsg() {
-	logger.Startup(color.Blue("\nDocs: ") + "https://min.io/docs/minio/linux/index.html")
+	var os string
+	switch runtime.GOOS {
+	case "darwin":
+		os = "macos"
+	case "linux":
+		os = "linux"
+	case "windows":
+		os = "windows"
+	default:
+		os = "linux"
+	}
+
+	logger.Startup(color.Blue("\nDocs: ") + fmt.Sprintf("https://min.io/docs/minio/%s/index.html", os))
```

Comment:
IMO it should just be...
```suggestion
	logger.Startup(color.Blue("\nDocs: ") + fmt.Sprintf("https://docs.min.io"))
```

Less likely to break in the future.